.gitignore

```diff
@@ -37,6 +37,9 @@ pip-delete-this-directory.txt
 htmlcov/
 .tox/
 .coverage
+# When running coverage testing 
+# it creates .coverage-username.123x* files
+.coverage.*
```

Comment:
Please remove this commit and send it in a separate PR, let's keep changes like this one clean and focused, unrelated changes make auditing and bug hunting harder.

Reply:
Sure